### PR TITLE
Fix Monitor Readme

### PR DESCRIPTION
**Pull request by LianwMS**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/monitor/resource-manager/readme.python.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ Generate all API versions currently shipped for this package
 
 ```yaml $(python) && $(multiapi)
 batch:
+  - tag: package-2019-10-17-preview-only
```
Comment (chenjianfei2017) [inline specification/monitor/resource-manager/readme.python.md:23]: need this one here: package-2019-10-preview?

Comment (LianwMS) [reply specification/monitor/resource-manager/readme.python.md:23]: why? I just change the tag name to follow the previous practice

Comment (chenjianfei2017) [reply specification/monitor/resource-manager/readme.python.md:23]: you changed tag name from Tag: 2019-10-17-preview-only to Tag: package-2019-10-preview in readme.md
so, in readme.python.md, tag here shoud use package-2019-10-preview defined in readme.md
pls duble confirm

Comment (LianwMS) [reply specification/monitor/resource-manager/readme.python.md:23]: Yes, you are correct. Just fix it

**Review by weidongxu-microsoft — APPROVED** [review]

**Pull request merged**